### DISCUSSION TITLE
function to supports RSA encryption

### DISCUSSION THREAD

**Discussion by zlgxzswjy** [projectdiscovery/nuclei-templates#7493] [Q&A]
I encountered a vulnerability in the system, which uses rsa encryption to encrypt the parameters during the login phase. During the vulnerability mining process, I already knew the public key used for encryption, but when I wrote the nuclei template, I couldn't find the calculation function related to rsa encryption. Is there any way i can implement this encryption process through the nuclei template

**Comment by zlgxzswjy** [comment]
The encryption process is implemented using JSEncrypt，and encryption scheme is RSAES-PKCS1-V1_5

**Comment by ehsandeep** [comment]
@zlgxzswjy all the helper function supported in nuclei is from https://github.com/projectdiscovery/dsl; I've created a [new issue](https://github.com/projectdiscovery/dsl/issues/45) to write / support a helper function for this, as of now it's not supported.

**Reply by zlgxzswjy** [reply]
Thank you for your timely support
